cleanup.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+OWNER=$(aws sts get-caller-identity | jq -r .Account)
+CUTOFF_DATE=$(date +%Y-%m-%d --date="-30 days")
+REGIONS=$(aws ec2 describe-regions | jq -r .Regions[].RegionName)
+DRY_RUN=true
+
+total_snapshots_removed=0
+total_snapshots_size=0
+
+while getopts ":f" opt; do
+    case ${opt} in
+        f)
+            DRY_RUN=false
+            ;;
+    esac
+done
+
+for region in ${REGIONS};
+do
+    echo " ➤ Scanning region ${region}"
+    TO_REMOVE=$(
+        aws ec2 describe-snapshots --owner-ids="${OWNER}" --region="${region}" | \
```

Comment:
do you think there is any possibility that `CUTOFF_DATE` could ever be null/empy/invalid and delete more than we expect?

Reply:
I don't think we can reach this point.